buildscripts/kokoro/unix.sh

```diff
@@ -70,10 +64,12 @@ popd
 
 LOCAL_MVN_TEMP=$(mktemp -d)
 # Note that this disables parallel=true from GRADLE_FLAGS
-./gradlew clean grpc-compiler:build grpc-compiler:uploadArchives $GRADLE_FLAGS -PtargetArch=x86_64 \
+./gradlew clean grpc-compiler:build grpc-compiler:uploadArchives $GRADLE_FLAGS \
   -Dorg.gradle.parallel=false -PrepositoryDir=$LOCAL_MVN_TEMP
 
-MVN_ARTIFACT_DIR="$PWD/mvn-artifacts"
-mkdir $MVN_ARTIFACT_DIR
-mv $LOCAL_MVN_TEMP/* $MVN_ARTIFACT_DIR
-rmdir $LOCAL_MVN_TEMP
+readonly MVN_ARTIFACT_DIR="${MVN_ARTIFACT_DIR:-$GRPC_JAVA_DIR/mvn-artifacts}"
+
+if [[ ! -d $MVN_ARTIFACT_DIR ]]; then
+  mkdir -p $MVN_ARTIFACT_DIR
```

Comment:
I think you can call `mkdir -p` unconditionally.

Reply:
removed `if`